libraries/rush-lib/src/logic/pnpm/PnpmfileConfiguration.ts

```diff
@@ -30,7 +30,7 @@ export interface IPnpmfileShimOptions {
 export class PnpmfileConfiguration {
   private _context: IPnpmfileContext | undefined;
 
-  public constructor(context: IPnpmfileContext) {
+  private constructor(context: IPnpmfileContext) {
```

Comment:
This is included in this PR as an extra precaution to make sure the `__non_webpack_require__` function in `PnpmfileShim` never gets called.